yatai-config.sample.yaml

```diff
@@ -20,7 +20,9 @@ s3:
   secure: true
 
 docker_registry:
-  server: https://index.docker.io/v1
+  bento_repository_name: <YOUR REPOSITORY NAME FOR BENTO>
```

Comment:
let's put a default name here?

Reply:
The repository name cannot have a default value when using docker hub, because the repository name contains the user's name, for example: `yetone/bento`